### PR TITLE
Fix getMetaInfo when another process holds the lock, Fixes #79

### LINKED ISSUE #79: Affinity is able to be acquired on cores locked by other processes
There's a bug in `AffinityLock.acquireLock` where locks held by other processes are not visible, meaning threads from separate processes are able to acquire a lock on the same core.

### PR DESCRIPTION
**Pull request by nicktindall**
This was something I noticed on Friday's troubleshooting session

### CODE REVIEW

File: affinity/src/main/java/net/openhft/affinity/common/ProcessRunner.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+import java.lang.management.RuntimeMXBean;
+import java.nio.charset.Charset;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ProcessRunner {
```
Comment (nicktindall) [inline affinity/src/main/java/net/openhft/affinity/common/ProcessRunner.java:19]: This should probably end up in some kind of `Chronicle-Test` shared library, there's a few things I'd like to see in a common shared test library

Comment (JerryShea) [reply affinity/src/main/java/net/openhft/affinity/common/ProcessRunner.java:19]: +1

**Review by JerryShea — APPROVED** [review]
nice!

File: affinity/src/main/java/net/openhft/affinity/common/ProcessRunner.java (hunk 1 above)
Comment (minborg) [reply affinity/src/main/java/net/openhft/affinity/common/ProcessRunner.java:19]: +1, A shared test library is precisely the thing we should do. I can think of a number of useful methods we could put in there including exhaustive test support with all permutations.

**Pull request merged**